examples/hello_worlds.rb

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env ruby
+
+require 'vedeu'
```

Comment:
Sorry, I'm not sure if I've miscommunicated here. The `fail Vedeu::VERSION` line was simply to allow you to compare the versions of Vedeu.

I've just built a Docker image which will allow me to test in a 'bare-bones' fashion (who knows what crazy workarounds are in my development environment! :sweat_smile:.